src/api/routers/shift.py

```diff
@@ -34,6 +35,10 @@ class ShiftCBV:
         status_code=HTTPStatus.CREATED,
         summary="Создать новую смену",
         response_description="Информация о созданной смене",
+        responses={
+            400: {"description": "Bad Request Response", "model": ErrorResponse},
+            403: {"description": "Forbidden Response", "model": ErrorResponse},

```

Comment:
Лучше создать словари для каждой ошибки, а то совсем много кода дублируется и проблематично будет поменять шаблон у всех роутеров.
```suggestion
            400: ERROR_TEMPLATE_FOR_400,
            403: ERROR_TEMPLATE_FOR_403,
```